src/timetell/importer.py

```diff
@@ -543,6 +555,273 @@
+        'constraints': """
+ALTER TABLE "{schemaname}"."{tablename}"
+    ADD CONSTRAINT fk_emp_id FOREIGN KEY (emp_id) REFERENCES "{schemaname}"."EMP"(emp_id),
+    ADD CONSTRAINT fk_act_id FOREIGN KEY (act_id) REFERENCES "{schemaname}"."ACT"(act_id),
+    ADD CONSTRAINT fk_prj_id FOREIGN KEY (prj_id) REFERENCES "{schemaname}"."PRJ"(prj_id),
+    -- ADD CONSTRAINT fk_cust_id FOREIGN KEY (cust_id) REFERENCES "{schemaname}"."CUST"(cust_id),
+    ADD CONSTRAINT fk_plan_request_id FOREIGN KEY (plan_request_id) REFERENCES "{schemaname}"."PLAN_REQUEST"(plan_request_id),
+    ADD CONSTRAINT fk_plan_task_id FOREIGN KEY (plan_task_id) REFERENCES "{schemaname}"."PLAN_TASK"(plan_task_id);
+        """
+    },
+    PLAN_REQUEST={
+        'create': """
+CREATE TABLE IF NOT EXISTS "{schemaname}"."{tablename}" (
+    plan_request_id integer PRIMARY KEY,
+    emp_id integer,
+    org_id integer,
+    skill_id integer,
+    fromdate date,
+    todate date,
+    "desc" character varying(50) collate pg_catalog."default",
+    info text,
+    hours double precision,
+    status integer,
+    owner_id integer,
+    changedate date,
+    status_alloc integer,
+    tag integer,
+    tagtype integer,
+    tagdate date,
+    plan_task_id integer,
+    changed_by integer,
+    status_date date,
+    days integer,
+    weekhours double precision,
+    status_allocinfo character varying(50),
+    rate double precision,
+    internalrate double precision
+)
+        """,
+        'constraints': """
+ALTER TABLE "{schemaname}"."{tablename}"
+    ADD CONSTRAINT fk_emp_id FOREIGN KEY (emp_id) REFERENCES "{schemaname}"."EMP"(emp_id),
+    ADD CONSTRAINT fk_org_id FOREIGN KEY (org_id) REFERENCES "{schemaname}"."ORG"(org_id),
+    ADD CONSTRAINT fk_plan_task_id FOREIGN KEY (plan_task_id) REFERENCES "{schemaname}"."PLAN_TASK"(plan_task_id);
+        """
+    },
+    PLAN_PRJ={
+        'create': """
+    CREATE TABLE IF NOT EXISTS "{schemaname}"."{tablename}" (
+        plan_prj_id integer PRIMARY KEY,
+        prj_id integer,
+        fromdate date,
+        todate date,
+        calcduration integer,
+        version integer,
+        tag integer,
+        tagtype integer,
+        tagdate date
+    )
+        """,
+    #         'constraints': """
+    # ALTER TABLE "{schemaname}"."{tablename}"
+    #     ADD CONSTRAINT fk_prj_id FOREIGN KEY (prj_id) REFERENCES "{schemaname}"."PRJ"(prj_id);
+    #         """
+        },
+    EMP_SKILLS={
+        'create': """
+CREATE TABLE IF NOT EXISTS "{schemaname}"."{tablename}" (
+    emp_skills_id integer PRIMARY KEY,
+    emp_id integer,
+    skill_id integer,
+    "order" integer,
+    "desc" character varying(50) collate pg_catalog."default",
+    perc double precision,
+    tag integer,
+    tagtype integer,
+    tagdate date
+)
+        """,
+        'constraints': """
+ALTER TABLE "{schemaname}"."{tablename}"
+    ADD CONSTRAINT fk_emp_id FOREIGN KEY (emp_id) REFERENCES "{schemaname}"."EMP"(emp_id);
+            """
+        },
```

Comment:
.... Yeah, this can't really be code reviewed. It seems like generated migration logic. Next time try separating from business logic.